irr/src/CIrrDeviceSDL.cpp

```diff
@@ -652,10 +652,17 @@ bool CIrrDeviceSDL::run()
 
 			irrevent.EventType = irr::EET_MOUSE_INPUT_EVENT;
 			irrevent.MouseInput.Event = irr::EMIE_MOUSE_MOVED;
-			MouseX = irrevent.MouseInput.X = SDL_event.motion.x;
-			MouseY = irrevent.MouseInput.Y = SDL_event.motion.y;
+
+			if (!SDL_GetRelativeMouseMode()) {
+				MouseX = irrevent.MouseInput.X = SDL_event.motion.x;
+				MouseY = irrevent.MouseInput.Y = SDL_event.motion.y;
+			} else {
+				MouseX = irrevent.MouseInput.X = MouseX + SDL_event.motion.xrel;
+				MouseY = irrevent.MouseInput.Y = MouseY + SDL_event.motion.yrel;
```

Comment:
does this make a difference on platforms other than Android? at least Multicraft Irrlicht also applies it on all platforms: https://github.com/MoNTE48/Irrlicht/blob/a2910941a0c64e58f294e3e4aa041fb6f7a1db9a/source/Irrlicht/CIrrDeviceSDL.cpp#L856-L865